secant/UI Components/CircularFrame/CircularFrame.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  CircularImageFrame.swift
+//  secant-testnet
+//
+//  Created by Adam Stener on 9/29/21.
+//
+
+import SwiftUI
+
+struct CircularFrame: View {
+    var body: some View {
+        GeometryReader { proxy in
+            let lineWidth = proxy.size.width * 0.07
+
+            Circle()
+                .stroke(lineWidth: lineWidth)
+                .foregroundColor(Asset.Colors.Onboarding.circularFrame.color)
+                // Add two points to the frame to properly mask edges
+                .frame(
+                    width: proxy.size.width - lineWidth + 2,
+                    height: proxy.size.height - lineWidth + 2,
+                    alignment: .center
+                )
+                // Update the offset to account for the 2 extra points
+                .offset(x: lineWidth / 2 - 1, y: lineWidth / 2 - 1)
+                .shadow(radius: 10)
+        }
+    }
+}
+
+struct OnboardingCircularFrame: View {
```

Comment:
It's used in the preview to prove that we we built underneath can in fact be animatible, and it will be used in the OnboardingScreen.  Much like the progress indicator view, that was only used in previews when that PR was first introduced, it will be used in the app once the OnboardingScreen comes together, this PR is just for reusable components, the next steps are to starting bringing all these components together.

Reply:
noted!